src/twitch/players/DesktopArticleVideoPlayer.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useRef } from 'react';
+import TwitchPlayerWrapper from 'twitch/players/shared/TwitchPlayerWrapper';
+import PlayerWrapper from 'twitch/players/shared/PlayerWrapper';
+import WDSVariables from '@fandom-frontend/design-system/dist/variables.json';
+import styled, { css, keyframes } from 'styled-components';
+// import UnmuteButton from 'jwplayer/players/DesktopArticleVideoPlayer/UnmuteButton';
+import useOnScreen from 'utils/useOnScreen';
+// import { ArticleVideoDetails } from 'jwplayer/types';
+// import CloseButton from 'jwplayer/players/shared/CloseButton';
+
+const DesktopArticleVideoTopPlaceholder = styled.div`
+	position: absolute;
+	width: 100%;
+	padding-top: 56.25%;
+	top: 0;
+	left: 0;
+	bottom: 0;
+	right: 0;
+	z-index: 2;
+`;
+
+const moveDownAnimation = (right: number, width: number) => keyframes`
+	from {
+		right: ${right}px;
```

Comment:
Can these props be passed down as `undefined` from `placeholderRef.current?.getBoundingClientRect();` ?